robokassa/migrations/0003_load_source_type.py

```diff
@@ -7,16 +7,10 @@
 class Migration(DataMigration):
 
     def forwards(self, orm):
-        "Write your forwards methods here."
-        # Note: Remember to use orm['appname.ModelName'] rather than "from appname.models..."
-        from django.core.management import call_command
-        from os.path import dirname, join
-        fixture_path = join(dirname(__file__), "payment.sourcetype.json")
```

Comment:
This file is not used anymore.

Reply:
https://github.com/100maketov/django-oscar-robokassa/pull/10/commits/e91d9baf6abb4f314e77c24a595bef629bd8602a

Reply:
https://github.com/100maketov/django-oscar-robokassa/pull/10/commits/61450455d6dd2e2cd093a341db3fa2383a054b83